javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,9 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  for (var i = values.length - 1; i >= 0; i--) {
+    if (typeof values[i] === 'string') {
+      values.splice(i, 1);
```

Comment:
Oh no!! You don't want to change the original array, unless it was the specific purpose of the function. Instead of taking values out of the original, you should create a new array with only the values that aren't strings